code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -80,9 +80,13 @@
 			user << "<span class='danger'>[src] buzzes and beeps.</span>"
 
 /mob/living/simple_animal/bot/cleanbot/process_scan(obj/effect/decal/cleanable/D)
+	. = list()
 	for(var/T in target_types)
 		if(istype(D, T))
-			return D
+			. += D
+	if(.)
+		return(pick(.))

```

Comment:
This was part of another PR #14612 , it must have ended up here accidentally.

Don't worry this was already fixed over there.